pilot/pkg/config/kube/crdclient/client.go

```diff
@@ -423,24 +422,14 @@ func (cl *Client) kind(r config.GroupVersionKind) (*cacheHandler, bool) {
 // knownCRDs returns all CRDs present in the cluster, with timeout and retries.
 func knownCRDs(crdClient apiextensionsclient.Interface) (map[string]struct{}, error) {
 	var res *crd.CustomResourceDefinitionList
-	b := backoff.NewExponentialBackOff()
```

Comment:
What's the reasoning behind this change?

Reply:
We will  retry at the upper level